### PR TITLE
A Gentle Introduction to Exponential Regression in Python

### LINKED ISSUE #7228: Getting Started with Growth Model in Pyhton
#### Proposed title of the article
Getting Started with Growth Model in Pyhton
#### Proposed article introduction
Growth models are common in scientific fields and have been developed and used
successfully for specific situations. Some of the situations where these models have shown success are in the description of how something grows with changes in the regressor variable, which is usually time. A life-time example where the growth model is used is modelling the time it takes until the business grows and other situations such as modelling the population growth with time. Growth models take the form:

$y = \frac{a}{(1 + be^{ −cx})}$,

where a , b and c are the constants of the model. At x = 0 , y = a/(1+b)

and as x → ∞ ,

y → a.

In this article, we will learn how to approximate the unknown parameters of the growth model using numerical methods such as the newton raphson algorithm and later state our model, which we will then use to make predictions. These activities will be carried out in python.

#### Key takeaways 
1. What's the Growth model
2. How to establish an optimization problem for the Growth model
3. How to determine the values of the unknown parameters by solving the lost function using the Newton Raphson method
4. How to make the prediction using the growth model
5. How to implement the growth model in python

#### Article quality
This article focuses on making the concept behind the growth model as clear as possible. We will focus on the key concepts and explore them in-depth. During our study, we will simplify concepts and make them easy to understand. This is because the content of this paper will base on a deep understanding of the topic. For the case study, we will provide a thoroughly explained code to make it easy for the learner to connect the theoretical part with the implementation session. Therefore, simplifying the concept, an in-depth approach to the study, and an easy-to-follow implementation session will make the entire content unique.
#### References
N/A

### PR DESCRIPTION
**Pull request by sumbaelvis**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #7228

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by lalith1403** [comment]
👋 Good afternoon - and welcome to the EngEd program.
Many sentences in this article appear to be taken from third-party sources. We can't publish any content that is not original or adequately cited/referenced. There are many free online tools where you can check this (such as quetext.com, and if you're uncertain about the definitions around plagiarism, this is an excellent resource to reference.
We're looking for unique/original content.

**Comment by lalith1403** [comment]
plagiarism check done ✔️

**Review by paulodhiambo — APPROVED** [review]

File: content/articles/growth-model-in-python/index.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+author: sumba-elvis
+date: 2022-04-30T00:00:00-12:14
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/growth-model-in-python/hero.jpg
+    alt: A Gentle Introduction to Exponential Regression in Python
+---
+
```
Comment (WanjaMIKE) [inline content/articles/growth-model-in-python/index.md:17]: Front matter inserted wrongly/incomplete

File: content/articles/growth-model-in-python/index.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+$$\Large = \sum_{i = 1}^{n}\left(y_i - \frac{a}{(1 + be^{ −cx})}\right)^2$$
+
+This is our optimization function.
+
+Now, the goal is to determine the values of the unknown coefficients, a, b, and c, that minimizes the above cost function. Since our cost function is quadratic, we minimize it by partially differentiating it with respect to each of the unknown parameters and equating the resulting equations to zero.
+
+If we do this, we should get:
+
+![image](engineering-education/growth-model-in-python/partial-derivatives.png)
```
Comment (WanjaMIKE) [inline content/articles/growth-model-in-python/index.md:60]: Is this link correct?

**Comment by WanjaMIKE** [comment]
@paulodhiambo I think the review on this PR was rushed. This is evident on the numerous errors still in the article. cc @sumbaelvis

**Comment by hectorkambow** [comment]
@sumbaelvis 
Be sure to see your live published article on the site (it should be live within 30 mins).

To give your article a boost in page engagement - share it across your social site (linkedin/twitter) and tag the EngEd program so we can also share it and give it some traction as well!

If you have not done so already - consider adding a skill section to your author page. This will go a long way for potential future employers.

**Pull request merged**